runtime/doc/lsp.txt

```diff
@@ -699,6 +699,24 @@ buf_detach_client({bufnr}, {client_id})          *vim.lsp.buf_detach_client()*
       • {bufnr}      (integer) Buffer handle, or 0 for current
       • {client_id}  (integer) Client id
 
+buf_inlay_hint_disable({bufnr})             *vim.lsp.buf_inlay_hint_disable()*
+    Disable inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}  (integer|nil) Buffer handle, or nil for current
+
+buf_inlay_hint_enable({bufnr})               *vim.lsp.buf_inlay_hint_enable()*
+    Enable inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}  (integer|nil) Buffer handle, or nil for current
+
+buf_inlay_hint_toggle({bufnr})               *vim.lsp.buf_inlay_hint_toggle()*
+    Toggle inlay hints for a buffer
+
+    Parameters: ~
+      • {bufnr}  (integer|nil) Buffer handle, or nil for current
+
```

Comment:
Do we really need 4 functions for this?

Reply:
Do you want only a toggle function?

Reply:
Aren't enable and disable necessary for scripting

Reply:
And toggle is necessary if you want a mapping to toggle

Reply:
done